syntax = "proto3";

import "test_import2.a.proto";
import "test_import.a.proto";
import "test_import.proto";
import "test_import2.proto";

package message1;

enum aslkfj {
	aaa = 0;
}
message c {
	message c1 {
		message c2 {
			message c3 {
				message c3 {
				}
			}
			c1.c2 kk = 1;
		}
	}
}
message InternalMessage {
	enum Request {
		Register = 0;
		DeRegister = 1;
		Heartbeat = 2;
		Search = 3;
		Response = 4;
	}
	enum Identity {
		Client = 0;
		Server = 1;
		ZooKeeper = 2;
	}
	message a1 {
		message a2 {
			message a3 {
				a2 kk = 33;
			}
		}
	}
	TTTT TTTT = 101003;
	InternalMessage.a1.a2 kk = 33333;
	AAAA cbcbcbcbcbcbcbc = 1000;
	Identity send_identity_ = 1;
	Identity recv_identity_ = 2;
	string service_name_ = 3;
	string method_name_ = 4;
	Request request_type_ = 6;
	string server_ip_ = 9;
	string server_port_ = 10;
	TTTT ABCDEFG = 100000;
	InternalMessage.Identity FFFFFFFFFF = 3333;
}
message c1 {
}
message AAAA {
}
message Imagine_Rpc {
}
message TTTT {
	InternalMessage.Identity taaa = 2;
}
